example/pubspec.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  audiomob:
+  audioadmob:
```

Comment:
why did you choose audiomob over audiomob name?

Reply:
to exclude any possible name conflicts with audiomob sdk